### PR TITLE
Pin url_launcher package to prevent major version upgrade

### PR DESCRIPTION
**Pull request by aam**

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/update_packages.dart [hunk 1]
```diff
@@ -31,6 +31,7 @@ const Map<String, String> _kManuallyPinnedDependencies = const <String, String>{
   // TODO(amirh): unpin this.
   'test': '0.12.26',
   // Add pinned packages here.
+  'url_launcher': '1.0.3',
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/update_packages.dart]: you should give a comment, with a TODO, as with the 'test' package above or as in the example in the documentation above.

Comment (Hixie) [reply packages/flutter_tools/lib/src/commands/update_packages.dart]: (ideally the comment would link to the bug report explaining why it's pinned)

**Comment by Hixie** [comment]
RSLGTM

**Comment by aam** [comment]
I should have tried `flutter build apk` locally, but pinning just url_launcher was not sufficient.
Two more have to be pinned:
```
  'connectivity' : '0.1.1',
  'device_info': '0.0.5',
```

**Pull request merged**